plugins/package-managers/maven/src/test/kotlin/TychoTest.kt

```diff
@@ -245,3 +348,18 @@ private fun injectCliMock(
 
     return cli
 }
+
+/**
+ * Return a reference to the Maven pom file in this folder.
+ */
+private fun File.pom(): File = resolve("pom.xml")
```

Comment:
How about making this an extension property instead for a more succinct syntax at the caller side?